dev/NavigationView/NavigationView_InteractionTests/FocusBehaviorTests.cs

```diff
@@ -243,14 +243,24 @@ public void CanDoSelectionChangedOfItemTemplate()
         {
             using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "NavigationView ItemTemplate Test" }))
             {
-                // Go to the navview items.
+                // Set focus inside page
+                var focusButton = FindElement.ByName("FocusAnchorButton");
+                focusButton.SetFocus();
```

Comment:
Why is this now needed?

Reply:
Ah i see now.

---
In reply to: [447890075](https://github.com/microsoft/microsoft-ui-xaml/pull/2771#discussion_r447890075) [](ancestors = 447890075)